Formula/f/flann.rb

```diff
@@ -21,6 +21,10 @@ class Flann < Formula
   depends_on "hdf5"
   depends_on "lz4"
 
+  on_macos do
+    depends_on "lz4"
+  end
```

Comment:
Lost a revision bump.

Also duplicate dependency
```suggestion
```

Reply:
Fixed. I did not double check, sorry.
Is there a way to avoid loosing revisions when rebasing (without manually reviewing the PR?)

Reply:
You can do `git log --oneline -16` each time you rebase to double-check quickly you haven't lost anything. If you have then you'll have to manually check which one disappeared though.